docs/operations/default_seccomp_profile_seed.md

```diff
@@ -1,5 +1,5 @@
 ---
-description: Enable the use of `RuntimeDefault` as the default seccomp profile through `spec.kubernetes.kubelet.seccompDefault`
+title: Default Seccomp Profile for Seeds
```

Comment:
In https://github.com/gardener/gardener/pull/9799#pullrequestreview-2147430906 I suggested 3 PRs where the 2nd PR contains only doc changes. Now a reviewer has to pay attention to multiple things at the same time - content changes and to section changes.